src/modules/removeTodo.js

```diff
@@ -0,0 +1,12 @@
+import getTodo from './getTodo.js';
+// import setTodo from "./setTodo.js";
```

Comment:
- You made a great job so far , but you committed old pieces of code as inline comments. They will make your project look messy. Kindly improve that by deleting these line 👌